### PR TITLE
update comosdb container names

### PR DESCRIPTION
**Pull request by wdavidcamden**
### Description

update comosdb container names

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: helm/config/dev/appsettings.Development.json [hunk 1]
```diff
@@ -56,57 +54,66 @@
...
       "VectorizationSource": {
-        "Endpoint": "https://ncus-qsl-openai-poc.openai.azure.com/openai/deployments/text-embedding-ada-002/embeddings?api-version=2023-07-01-preview",
+        "Endpoint": "https://ncus-qsl-openai-poc.openai.azure.com/openai/deployments/text-embedding-ada-002/embeddings?api-version=2023-07-01-preview" //disable s
       }
     },
     "SpecializationIndexes": [
       {
         "IndexName": "david-probate-files",
         "QueryType": "vector_simple_hybrid", //Supported Options: simple, semantic, vector, vector_simple_hybrid, vector_semantic_hybrid
         "GroupMemberships": [
```
Comment (dhaselhan) [inline helm/config/dev/appsettings.Development.json:70]: I think these can be deleted, as now the GroupMemberships comes from the Specialization when its created

**Review by dhaselhan — DISMISSED** [review]
LGTM 👍

File: helm/config/dev/appsettings.Development.json (hunk 1 above)
Comment (wdavidcamden) [reply helm/config/dev/appsettings.Development.json:70]: removed now

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**